packages/app/src/logEntryProcessing.ts

```diff
@@ -111,6 +111,9 @@ export function lambdaRequestLogData(
 					lambdaStats: stats,
 				};
 			}
+			default: {
+				return undefined;
+			}
```

Comment:
The `tsconfig.json` at the root is now quite strict. Changes to this file resolves these errors:

```
error TS7030: Not all code paths return a value.
```